src/libs/SearchUIUtils.ts

```diff
@@ -861,69 +882,180 @@ function isCorrectSearchUserName(displayName?: string) {
-                return ROUTES.SEARCH_ROOT.getRoute({query});
-            },
-        },
-        {
-            translationPath: 'common.tasks',
-            type: CONST.SEARCH.DATA_TYPES.TASK,
-            icon: Expensicons.Task,
-            getRoute: (policyID?: string) => {
-                const query = buildCannedSearchQuery({type: CONST.SEARCH.DATA_TYPES.TASK, status: CONST.SEARCH.STATUS.TASK.ALL, policyID});
-                return ROUTES.SEARCH_ROOT.getRoute({query});
-            },
+            translationPath: 'common.explore',
+            menuItems: [
+                {
+                    translationPath: 'common.expenses',
+                    type: CONST.SEARCH.DATA_TYPES.EXPENSE,
+                    icon: Expensicons.Receipt,
+                    getSearchQuery: (policyID?: string) => {
+                        const queryString = buildCannedSearchQuery({policyID});
+                        return queryString;
+                    },
+                },
+                {
+                    translationPath: 'common.reports',
+                    type: CONST.SEARCH.DATA_TYPES.EXPENSE,
+                    icon: Expensicons.Document,
+                    getSearchQuery: (policyID?: string) => {
+                        const queryString = buildCannedSearchQuery({groupBy: CONST.SEARCH.GROUP_BY.REPORTS, policyID});
+                        return queryString;
+                    },
+                },
+                {
+                    translationPath: 'common.chats',
+                    type: CONST.SEARCH.DATA_TYPES.CHAT,
+                    icon: Expensicons.ChatBubbles,
+                    getSearchQuery: (policyID?: string) => {
+                        const queryString = buildCannedSearchQuery({type: CONST.SEARCH.DATA_TYPES.CHAT, status: CONST.SEARCH.STATUS.CHAT.ALL, policyID});
+                        return queryString;
+                    },
+                },
+            ],
         },
     ];
 
-    if (canSendInvoice(allPolicies, email) || hasInvoiceReports()) {
-        typeMenuItems.push({
-            translationPath: 'workspace.common.invoices',
-            type: CONST.SEARCH.DATA_TYPES.INVOICE,
-            icon: Expensicons.InvoiceGeneric,
-            getRoute: (policyID?: string) => {
-                const query = buildCannedSearchQuery({type: CONST.SEARCH.DATA_TYPES.INVOICE, status: CONST.SEARCH.STATUS.INVOICE.ALL, policyID});
-                return ROUTES.SEARCH_ROOT.getRoute({query});
-            },
-        });
-    }
+    // We suggest specific filters for users based on their access in specific policies. Show the todo section
+    // only if any of these items are available
+    const showSubmitSuggestion = getAllSharedPolicies().length > 0;
+    const showApproveSuggestion = getApproverPolicies().length > 0;
+    const showPaySuggestion = getAdminPolicies().length > 0;
+    const showExportSuggestion = getExporterPolicies().length > 0;
```

Comment:
I left a comment above about this, but I think we should get all policies from the view and pass it as a param to this function so it updates if the policy list updates